spec/feature/home/index_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+feature 'User visited the home page' do
+  context 'with role: whitelisted_user' do
+    given(:user) { create(:user, role: :admin) }
+
+    background do
+      logged_in_as user
+      visit root_path
+    end
+
+    scenario 'and saw greeting' do
+      expect(page).to have_content("Hello, #{user.name}")
+    end
+  end
+
+  context 'with role: user' do
+    given(:user) { create(:user) }
+
+    background do
+      logged_in_as user
+      visit root_path
+    end
+
+    scenario 'and saw greeting and CLA link' do
+      expect(page).to have_content("Hello, #{user.name}")
+      expect(page).to have_link("Contributor's License Agreement")
+    end
+
+    scenario 'and clicked CLA link' do
+      click_on("Contributor's License Agreement")
+
+      expect(page).to have_content("Hello, #{user.name}, please read:")
+      expect(page).to have_content("Contributor's License Agreement")
+      expect(page).to have_button("Please sign in with Github to sign")
```

Comment:
Why do you see a "please sign in with Github" when you're already signed in?
This makes no sense!